src/it/scala/org/broadinstitute/dsde/rawls/WorkspaceSimulation.scala

```diff
@@ -269,4 +281,99 @@ class WorkspaceSimulation extends FlatSpec with WorkspaceApiService with EntityA
   it should "create + list many workspaces" in {
     // TODO
   }
+
+  it should "copy a method config from the method repo" in {
```

Comment:
My inclination would be to move these tests elsewhere - ```WorkspaceSimulation``` should be for load testing, using data generated by ```WorkspaceGenerator```, and everything in it should be a ```timedOperation```. To what extent do these overlap with the unit tests added to ```WorkspaceApiServiceSpec``` ?